jib-core/src/test/java/com/google/cloud/tools/jib/builder/BuildConfigurationTest.java

```diff
@@ -198,7 +179,7 @@ public void testBuilder_missingValues() {
 
     } catch (IllegalStateException ex) {
       Assert.assertEquals(
-          "base image is required but not set, target image is required but not set, and main class is required but not set",
```

Comment:
Hmm I think we should keep a test for the `default` case of having 3 or more error messages. Or we can just delete that `default` case logic since we only have two possible error messages at the moment (missing target and missing base).